.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+github:
+- .github/**/*
+
+helper:
+- helper/**/*
+
+technolgy/sev:
+- internal/shim-sev/**/*
+
+technolgy/sgx:
+- internal/shim-sgx/**/*
+
+technology/wasm:
+- internal/wasmldr/**/*
+
+src:
+- src/**/*
+
+backend:
+- src/backend/**/*
+
+cli:
+- src/cli/**/*
+
+protobuf:
+- src/protobuf/**/*
+
+workldr:
+- src/workldr/**/*
+
+tests/bin:
+- tests/bin/**/*
+
```

Comment:
I would recommend simply labeling all of the content in `tests/**/*`  as `testing`.

Reply:
Sure thing!!